src/Scene.cpp

```
@@ -3,70 +3,109 @@
+  } else {
+    auto now = std::chrono::steady_clock::now();
+    auto duration =
+        std::chrono::duration_cast<std::chrono::seconds>(now - _lastMovement);
+    if (duration.count() > _updateQuality && !_isHighQuality) {
+      _isHighQuality = true;
     }
   }
-  file.close();
 }
 
 void Raytracer::Scene::render() {
```

Cppcheck (2.10-2)

[style] The function 'render' is never used.

(unusedFunction)